README.md

```diff
@@ -1,17 +1,21 @@
 ## sell-my-services
 
-We help people to offer their services to others easily and quickly. SellMyServices is a platform where a seller can publish his
-services. These services then can be searched by seekers in their nearby locations. 
+An open source content search tool. This is the backend repository of https://sellmyservices.in.
+To know more about product please check android repository https://github.com/gawdsnitkkr/sell-my-services-android
 
-#### How does it work?
+### Android app link
+[![Android App](./screenshots/google-play.png)](https://play.google.com/store/apps/details?id=me.varunon9.sellmyservices)
 
-1. A seller is a person who offers some service e.g. Photographer, Carpenter, Cook, Teacher, Doctor, Freelancer etc.
-2. Seller register his services on our platform.
-3. A seeker is a person who is in search of some service(s) e.g. a person looking to hire cook.
-4. Seekers search services on our platform.
-5. SellMyServices list all the nearby sellers who are offering services searched by seeker. 
-6. Seeker then can browser seller's services, their ratings, location etc and if interested can directly chat to sellers.
-7. After availing a service, a seeker can share feedback and rating to corresponding seller on our platform.
+#### The core idea
+
+SellMyServices is a platform to match `service seekers` to `service sellers`. A normal flow is-
+
+1. User installs our [Android app](https://play.google.com/store/apps/details?id=me.varunon9.sellmyservices)
+2. From the MainActivity he searches for services of his interset. Let's say he is looking for a `freelancer`.
```

Comment:
from MainActivity?

Reply:
I'll update this to MainScreen

Reply:
We can just remove screen, and `interest`

Reply:
2. In the app he searches for some services. Let's say he is looking for a `freelancer`. ??

Reply:
Sounds good.